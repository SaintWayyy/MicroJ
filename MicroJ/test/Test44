/* Test global variable */

int counter : = 0;
bool flag : = true;
double data : = 3.14;

int main(){
    while ( counter < 5 && flag ){
        counter = counter + 1;
        data = data * 2.0;
        if (data == 4.0 * 3.14) {
            flag = false;
        }
    }
    print(counter);
    /* The result should be 2 */
    print(flag);
    /* The result should be 0 */
    print(data);
    /* The result should be 12.560000 */
}