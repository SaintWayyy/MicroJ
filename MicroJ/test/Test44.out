2
0
12.560000
